### ISSUE TITLE
Can't view large np array in Spyder

### ISSUE THREAD

**Issue by catruck** [spyder-ide/spyder#8885]
### What steps will reproduce the problem?

Hello, 

It seems to be a problem when I try to visualise a large np arry.
For ex:

array of dim 3,10 can be viewed
but array of dim 30,100 get an error.

Thanks

### Traceback
```python-traceback
  File "C:\ProgramData\Anaconda3\envs\rec\lib\site-packages\spyder\widgets\variableexplorer\collectionseditor.py", line 487, in createEditor
    if not editor.setup_and_check(value, title=key, readonly=readonly):
  File "C:\ProgramData\Anaconda3\envs\rec\lib\site-packages\spyder\widgets\variableexplorer\arrayeditor.py", line 620, in setup_and_check
    self.data.flags.writeable = True
ValueError: cannot set WRITEABLE flag to True of this array
```

## Versions

* Spyder version: 4.0.0b1 
* Python version: 3.6.5
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
pyflakes >=0.6.0  :  2.1.0 (OK)
pycodestyle >=2.3 :  2.5.0 (OK)
pygments >=2.0    :  2.3.1 (OK)
sphinx >=0.6.6    :  1.8.4 (OK)
rope >=0.9.4      :  0.12.0 (OK)
jedi >=0.11.0     :  0.13.3 (OK)
nbconvert >=4.0   :  5.4.1 (OK)
pandas >=0.13.1   :  0.24.1 (OK)
numpy >=1.7       :  1.16.2 (OK)
sympy >=0.7.3     :  1.3 (OK)
cython >=0.21     :  0.29.5 (OK)
qtconsole >=4.2.0 :  4.4.3 (OK)
IPython >=4.0     :  7.3.0 (OK)
matplotlib >=2.0.0:  3.0.2 (OK)
pylint >=0.25     :  2.2.2 (OK)
```

**Comment by catruck** [comment]
![arr_problem](https://user-images.githubusercontent.com/25405157/53797323-9f05f200-3f3e-11e9-9e13-3ac28e30e1cc.PNG)

**Comment by ccordoba12** [comment]
This error was fixed in our latest stable version (3.3.3). However, that fix was not available in beta1, so you have two choices:

1. Wait until beta2 (we don't know when that's going to be ready).
2. Patch Spyder source code yourself to avoid the error. The fix is in PR #8632.
